### PR TITLE
Occupational Corruption Device

### PR DESCRIPTION
**Pull request by Orchian**
## About The Pull Request
Adds a new role-specific traitor item for the Head of Personnel and the Quartermaster that triggers a bureaucratic error then self-deletes.

## Why It's Good For The Game
Adds an additional method for sewing havoc upon the station at large.

## Changelog
:cl:
add: Adds a new traitor item for HoP and QM that sews non-immediate havoc upon the station
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/ocd.dm [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/obj/item/devices/ocd_device
+    //HOW HAVE I SURVIVED THIS LONG WITHOUT BRAINCELLS?!
+	name = "Occupational Corruption Device"
+	desc = "When you need to make the lives of new-hires that much more confusing, think OCD."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "gangtool-white"
+
+/obj/item/devices/ocd_device/attack_self(mob/user)
+    var/datum/round_event/bureaucratic_error/event = new(queue_event = FALSE) //God I'm fucking stupid
+    event.start() //I'm REALLY fucking stupid
```
Comment (LemonInTheDark) [inline code/game/objects/items/devices/ocd.dm:10]: No need to self deprecate in comments, the pr is working, you're not dumb

Comment (Orchian) [reply code/game/objects/items/devices/ocd.dm:10]: Lemon, the self-deprecation is me forgetting dumb shit or being too dumb to be able to dissect it myself

**Comment by wesoda25** [comment]
It shouldn't delete itself, that makes it too easy on the traitor. For stuff like the crab phone, evidence is left over.

File: code/game/objects/items/devices/ocd.dm [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+	name = "Occupational Corruption Device"
+	desc = "When you need to make the lives of new-hires that much more confusing, think OCD."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "gangtool-white"
+
+/obj/item/devices/ocd_device/attack_self(mob/user)
+    var/datum/round_event/bureaucratic_error/event = new(queue_event = FALSE) //God I'm fucking stupid
+    event.start() //I'm REALLY fucking stupid
+    deadchat_broadcast("<span class='bold'> An OCD has been activated! </span>")
+    qdel(src)
```
Comment (TiviPlus) [inline code/game/objects/items/devices/ocd.dm:12]:
```suggestion
    qdel(src)

```
whiteline, also  why does the uplink_items.dm file have 4020 changed lines?

Comment (Orchian) [reply code/game/objects/items/devices/ocd.dm:12]: I have literally no clue why

**Pull request closed without merging**